src/picongpu/include/plugins/adios/ADIOSWriter.hpp

```diff
@@ -508,6 +506,10 @@ class ADIOSWriter : public ILightweightPlugin
             mpi_pos = gc.getPosition();
             mpi_size = gc.getGpuNodes();
 
+            /* if number of aggregators is not set we use all mpi process as aggregator*/
+            if( mThreadParams.adiosAggregators == 0 )
+               mThreadParams.adiosAggregators=mpi_size.productOfComponents();
```

Comment:
I see one problem for the case that `adiosOST` and `adiosAggregators` are set to default. Than the default configuration is: `1 OST and 3 aggregators`. (e.g. on our system *hypnos*) This always results in bad write performance if we use the default settings.
Aggregators are intresting for `10k+` cores, I am not sure if we can leave aggregators unset on small systems (<5k write processes).